extensions/frontend-tiles/src/integration-test/TestContext.ts

```diff
@@ -0,0 +1,57 @@
+import { AccessToken } from "@itwin/core-bentley";
+import { getAccessTokenFromBackend } from "@itwin/oidc-signin-tool/lib/cjs/frontend";
+
+export class TestContext {
+  private _accessToken: AccessToken | undefined;
+  private static _instance?: TestContext = undefined;
+
+  private constructor() { }
+
+  public static async instance(): Promise<TestContext> {
+    if (this._instance === undefined) {
+      this._instance = new TestContext();
+      await this._instance.initialize();
+    }
+    return this._instance;
+  }
+
+  public getAccessToken(): AccessToken | undefined {
+    return this._accessToken;
+  }
+
+  private async initialize() {
+    const urlPrefix = process.env.IMJS_URL_PREFIX || "";
+    const oidcConfig = {
+      clientId: process.env.MES_OIDC_CLIENT_ID || "",
```

Comment:
lets make sure this env var follows the same prefix like all the other env vars

Reply:
these env variables have now been changed to start with IMJS_MES_INTEGRATION so that they still begin with the IMJS prefix, but they are still also specified as part of the mes integration test so that there is no accidental value mix up when running these in a pipeline.